packages/api/server.js

```diff
@@ -23,21 +23,25 @@ const GetTestHandler = require('./src/get-test.js');
 const GetExportHandler = require('./src/get-export.js');
 const client = require('./src/firestore.js');
 const auth = require('./src/auth.js');
+const { InvalidParameterError } = require('./lib/errors');
 
 const { v4 } = require('uuid');
 
 const cors = require('cors');
 
-global.headCollection = process.env.HEAD_COLLECTION || 'testing/main/repos';
+global.headCollection = 'production/main/repos';
```

Comment:
FYI, This will/must be reverted **before merging**.
Access to the production collection is necessary to test the Button in staging.

Reply:
👍 let's not forget to revert this.